src/Umbraco.Core/Persistence/DatabaseSchemaHelper.cs

```diff
@@ -90,20 +138,54 @@ internal void CreateDatabaseSchemaDo(IMigrationEntryService migrationEntryServic
             _logger.Info<Database>("Finalized database schema creation");
         }
 
+        /// Creates a new table in the database based on the type of <typeparamref name="T"/>.
+        /// 
+        /// If <typeparamref name="T"/> has been decorated with an <see cref="TableNameAttribute"/>, the name from that
+        /// attribute will be used for the table name. If the attribute is not present, the name
+        /// <typeparamref name="T"/> will be used instead.
+        /// 
+        /// If a table with the same name already exists, the <paramref name="overwrite"/> parameter will determine
+        /// whether the table is overwritten. If <c>true</c>, the table will be overwritten, whereas this method will
+        /// not do anything if the parameter is <c>false</c>.
+        /// <typeparam name="T">The type representing the DTO/table.</typeparam>
+        /// <param name="overwrite">Whether the table should be overwritten if it already exists.</param>
         public void CreateTable<T>(bool overwrite)
            where T : new()
         {
             var tableType = typeof(T);
             CreateTable(overwrite, tableType);
         }
 
+        /// <summary>
+        /// Creates a new table in the database based on the type of <typeparamref name="T"/>.
+        /// 
```

Comment:
Remove blank line [142](https://github.com/umbraco/Umbraco-CMS/pull/3205/files#diff-140ac76b9076c9248a612ce2aacb042dR142) & [146](https://github.com/umbraco/Umbraco-CMS/pull/3205/files#diff-140ac76b9076c9248a612ce2aacb042dR146) as well may be? Has this been introduced purposefully?

Reply:
Yes, I added the extra lines by purpose. Whether it makes sense may depend on the tool used to generate the reference documentation. @nul800sebastiaan do you have some hints here?

As we don't want long lines in the code, it's normal to add a single line break, which then is just interpreted as regular white space. Some tools will treat a double line break similar to Markdown, making it a paragraph (which was my intention).

None of this do however affect how it's shown in Visual Studio, as it's just shown as a single line. It appears that Visual Studio won't even honor `<br />` elements in the description.